mmeds/validate.py

```diff
@@ -161,30 +161,47 @@ def check_string_column(self, column):
 
     def check_lengths(self, column):
         """ Checks that all entries have the same length in the provided column """
-        length = len(column[0])
+        try:
+            length = len(column[0])
+        except TypeError:
+            length = 0
+            err = '{}\t{}\tLength Error: Column {} had an unexpected length'
+            self.errors.append(err.format(-1, self.col_index, column.name))
         for i, cell in enumerate(column[1:]):
-            if not len(cell) == length:
+            try:
+                cell_length = len(cell)
+            except TypeError:
+                cell_length = 0
+                err = '{}\t{}\tLength Error: Cell {} had an unexpected length in column {}'
```

Comment:
If you got a type error, did the cell have an unexpected length or was it an unexpected type?